docs/02-connect/04-eon_block_explorer.md

```diff
@@ -0,0 +1,584 @@
+- **(Receiving) Address** - The receiving address associated with the transfer transaction. Click the address receiving asset to **Address Detail** page.
+
+## Gobi Testnet Block Explorer Features
+
+The Gobi Testnet Block Explorer gives you the ability to search the blockchain for transactions, addresses, and statistics, including the customized features of:
+
+- Backward Transfer
+- Forward Transfer
+- Fee Payment
+- Backward Transfer
+
+
+Typically, when a user decides to move ZEN from a sidechain back to the mainchain (to either send ZEN to another sidechain or to cash out, for example) the backward transfer transaction is deployed. Backward transfers are initiated on EON (EVM sidechain) as special transactions batched in withdrawal certificates and propagated by EON nodes to the mainchain.
+
+The Gobi Testnet Explorer highlights the Backward Transfer transaction:
+
+<p>
+<img src={require("/img/docs/blockscout/gobi-exp-backward.png").default} alt="backward-transaction" width="900" height="700" />
+</p>
+
+Key information includes:
+
+- **Transaction Status** - Success or Error: (Awaiting internal transaction for reason)
+- **Transaction ID (hash)** - The unique identifier that validates the transaction. Click to **Transaction Detail** page.
+- **(Sending) Address** - The sending address of the transaction. Click the address receiving asset (link) to **Address Detail** page.
+- **Amount of tZEN** - The amount of tZEN transferred in the transaction.
+- **Transaction Fee** - The amount charged to process the transaction.
+- **Block Number** - The number assigned to the block.
+- **Elapsed timer for the block** - A timer that displays the elapsed time since block creation period.
+
+## Forward Transfer
+
+When a user requests for tZEN from the Horizen Faucet, the forward transfer transaction is initiated. A forward transfer moves assets from the mainchain to the sidechain. These transactions, more specifically the transaction outputs, are unspendable on the mainchain, but include some metadata so they are redeemable on EON (EVM sidechain). 
+It is the responsibility of EON nodes to monitor the mainchain for incoming transactions, and include them in EON.
+
+The Gobi Testnet Explorer highlights the Forward Transfer transaction:
+
+<p>
+<img src={require("/img/docs/blockscout/gobi-exp-forward.png").default} alt="forward-transaction" width="900" height="700" />
+</p>
+
+Key information includes:
+
+- **Transaction Status** - Success or Error: (Awaiting internal transaction for reason)
+- **Transaction ID (hash)** - The unique identifier that validates the transaction.
+- **(Receiving) Address** - The recipient address of the transaction.
+- **Amount of tZEN** - The amount of tZEN transferred in the transaction.
+- **Block Number** - The number assigned to the block.
+- **Elapsed timer for the block** - A timer that displays the elapsed time since block creation period.

```

Comment:
This is different wording than the definition previously. We should make sure that all definitions are uniformed if they will be repeated on the page

Reply:
made corrections